modules/ROOT/content-nav.adoc

```diff
@@ -58,8 +58,9 @@
 ** xref:introspector.adoc[Introspector]
 
 ** xref:migration/index.adoc[Migration guide]
-*** xref:migration/authorization.adoc[]
-*** xref:migration/ogm.adoc[]
+*** xref:migration/4.0.0/index.adoc[4.0.0 Guide]
```

Comment:
I guess it would be easier to just make it 4, like we currently have on the website (e.g. https://neo4j.com/docs/graphql/4/), but then the folder name also needs to be changed alongside these links.